explore/src/main/scala/explore/targets/TargetSummaryTable.scala

```diff
@@ -155,11 +175,56 @@ object TargetSummaryTable extends TableHooks:
             table,
             striped = true,
             compact = Compact.Very,
-            tableMod = ExploreStyles.ExploreTable,
+            tableMod = ExploreStyles.ExploreTable |+| ExploreStyles.ExploreSelectableTable,
             headerCellMod = headerCell =>
               columnClasses
                 .get(headerCell.column.id)
                 .orEmpty |+| ExploreStyles.StickyHeader,
+            rowMod = row =>
+              TagMod(
+                ExploreStyles.TableRowSelected.when_(row.getIsSelected()),
+                ^.onClick --> Callback.log(s"cmd ${isHotkeyPressed("cmd")}") *> Callback {
+                  // If cmd is pressed add to the selection
+                  if (!isCmdCtrlPressed) table.toggleAllRowsSelected(false)
+                  if (isShiftPressed) {
+                    // If shift is pressed extend
+                    val selectedRows = table.getSelectedRowModel().rows.toList.sortBy(_.id)
+                    val allRows      = table.getRowModel().rows.toList.sortBy(_.id)
```

Comment:
Does this work if they have sorted the table by something other than ID?

Reply:
Good question, let me review. It is also a question for generalization. can we assume rows have an id?

Reply:
You'd have to check the current table implementation, but I seem to remember the old table having an id. You could assign it, otherwise it would assign one itself. But, I may be thinking of a different table.

Reply:
All rows have an id. From the doc for `getRowId()`:

`This optional function is used to derive a unique ID for any given row. If not provided the rows index is used (nested rows join together with . using their grandparents' index eg. index.index.index)`

Reply:
It seems like that would allow us to generalize the selection process.

Reply:
Ok, I added a fix that relies on the position of the selected rows rather than the id and it seems to work

Reply:
Does it work with virtualized tables?

Reply:
🙉